radio/src/gui/480x272/zone.h

```diff
@@ -0,0 +1,54 @@
+
+#ifndef _ZONE_H_
+#define _ZONE_H_
+
+#include <inttypes.h>
+#include <string.h>
+
+#define LEN_ZONE_OPTION_STRING         8
+
+#define OPTION_VALUE_UNSIGNED(x)    setZoneOptionValue((uint32_t)x)
```

Comment:
I see no issue with the solution I found and was removed by this commit:
https://github.com/opentx/opentx/commit/82dce3ee601049cf548ff4518902ab8e8fa62a0b